src/browser/topbar.ts

```diff
@@ -134,8 +137,8 @@ export const topbar = {
     }
     ;(function loop() {
       if (topbar.progress('+.1') >= 1) {
-        canvas.style.opacity -= 0.05
-        if (canvas.style.opacity <= 0.05) {
+        canvas.style.opacity = `${Number(canvas.style.opacity) - 0.05}`
+        if (Number(canvas.style.opacity) <= 0.05) {
```

Comment:
Frankly, I like how it was before :)

Previously this file was in the `vendor` folder, which I treat as "it's not our code, so we don't apply linters and rules for it, it just works as-is". Because of that - I feel more comfortable keeping things as-is, and making this PR a 1-liner - only adding the accessibility thing.

BTW, on accessibility. We don't currently use, and AFAIK don't plan to use topbar in production, because, for example, its UI is arbitrary/colorful and not on brand. It's only purpose is to aid in "developer mode" (similar to RedDot). Which would even be exempt from accessibility requirements. But I don't mind adding a line so accessibility audits don't scream.

Reply:
> Previously this file was in the vendor folder, which I treat as "it's not our code, so we don't apply linters and rules for it, it just works as-is". Because of that - I feel more comfortable keeping things as-is, and making this PR a 1-liner - only adding the accessibility thing.

Makes sense. Updated the PR 👍